config/main.py

```diff
@@ -1667,6 +1673,65 @@ def load_port_config(config_db, port_config_path):
                 port_name), display_cmd=True)
     return
 
+
+def load_golden_config(golden_config_path):
+    if not os.path.isfile(golden_config_path):
+        return
```

Comment:
This check should be outside the function. It is caller's responsible to treat missing file as failure or silently skipping.